### PR TITLE
Update documentation for Drone 0.8.x

### PR DESCRIPTION
**Pull request by lkiesow**
When trying out Drone, I found it really hard to start because of the
basic installation guides seem to not work anymore for Drone >= 0.8.
Especially the new(?) `agent` container did cost me quite some time.

This patch updates the documentation to work with Drone 0.8. I just
tried out Drone today, hence no guarantees about about correctness, but
making these changes, I was able to deploy Drone without problems.

### CODE REVIEW

File: themes/drone/layouts/shortcodes/version.html [hunk 1]
```diff
@@ -1 +1 @@
-0.7
\ No newline at end of file
+0.8.1
```
Comment (tboerger) [inline themes/drone/layouts/shortcodes/version.html]: 0.8 instead of 0.8.1

Comment (lkiesow) [reply themes/drone/layouts/shortcodes/version.html]: I can change that, but 0.8.1 is the latest version and changing that will make users pull an old docker container. What is the intention behind that?

Comment (jmccann) [reply themes/drone/layouts/shortcodes/version.html]: `0.8` tag will be updated with all patch releases.  So `0.8` right now **is** `0.8.1` and if `0.8.2` is released then `0.8` will become `0.8.2`.

So it future proofs people to latest `0.8` releases which shouldn't have breaking changes but only bug fixes.

Comment (lkiesow) [reply themes/drone/layouts/shortcodes/version.html]: Ok, did not know that. Thanks for the explanation. Will adjust the docs now…

Comment (lkiesow) [reply themes/drone/layouts/shortcodes/version.html]: done

**Pull request merged**